packages/app/components/jsonschemaform/jsonschemaform.tsx

```diff
@@ -7,8 +14,115 @@ import { useEffect, useRef } from 'react'
 import fields from './fields/'
 import templates from './templates/'
 import widgets from './widgets/'
-import validator from '@rjsf/validator-ajv8'
-import type { FormValidation } from '@rjsf/utils'
+
+class LargeModelForm<
```

Comment:
Hacky shenanigans...move the `change` event to be triggered on 'blur`. This form is only used if the accompanying model is marked as `largeModel`.